components/infieldprogresscircle/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@spectrum-css/infieldprogresscircle",
+  "version": "0.0.0",
+  "description": "The Spectrum CSS infieldprogresscircle component",
+  "license": "Apache-2.0",
+  "author": "Adobe",
+  "homepage": "https://opensource.adobe.com/spectrum-css/infieldprogresscircle",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/adobe/spectrum-css.git",
+    "directory": "components/infieldprogresscircle"
+  },
+  "bugs": {
+    "url": "https://github.com/adobe/spectrum-css/issues"
+  },
+  "main": "dist/index.css",
+  "files": [
+    "dist/*",
+    "CHANGELOG.md",
+    "package.json",
+    "stories/template.js",
+    "metadata/mods.md"
+  ],
```

Comment:
Would you double check the package.json for a few things? I used the regular progress circle package.json as a comparison, and I think we're missing a few things. 

- This `files` key looks like it's been updated with `"stories/*` and `"metadata/*`
- We're missing the `exports` block.
- We're missing the `devDependencies` block.
- We might have to double check the tokens packages declared for those peerDependencies and devDependencies.
- For the description up at the top of this file, we can separate out the component name. "The Spectrum CSS infield progress circle component" so that "infieldprogresscircle" isn't just one word.
- I also get a warning in my editor that "'peerDependencies' should be before 'keywords'" (not sure if you were getting that from eslint)